src/main/java/com/iota/iri/service/validation/TransactionValidator.java

```diff
@@ -398,6 +328,7 @@ public void updateStatus(TransactionViewModel transactionViewModel) throws Excep
      */
     private boolean quietQuickSetSolid(TransactionViewModel transactionViewModel) {
         try {
+            transactionSolidifier.addToSolidificationQueue(transactionViewModel.getHash());
```

Comment:
So you put in queue before it gets quickly solid?

Maybe better the other way around? Or maybe better to put it straight in `BroadcastQueue` if it gets quickly solid?